sql/analyze_test.go

```diff
@@ -54,7 +54,7 @@ func parseAndNormalizeExpr(t *testing.T, sql string) (parser.Expr, qvalMap) {
 		t.Fatalf("%s: %v", sql, err)
 	}
 	expr := q[0].(*parser.Select).Exprs[0].Expr
-	expr, err = parser.NormalizeExpr(expr)
+	expr, err = parser.NormalizeExpr(parser.EvalContext{}, expr)
```

Comment:
I don't see any benefit to one approach over the other. Switched to making it a method for consistency.